chat/views.py

```diff
@@ -79,7 +81,7 @@ def get(self, request, format=None):
             }
             messages.append(data)
 
-        response = {"peer_connected": peer_connected, "messages": messages}
+        response = {"peer_connected": peer_connected, "messages": messages, "peer_pubkey": peer_public_key}
```

Comment:
perfect!

Reply:
should I remove this? I see the response now includes it

Reply:
The response includes it because it is running the code in this PR :D